### ISSUE TITLE
Cannot see Role Binding in Rancher UI after changing LDAP System User

### ISSUE THREAD

**Issue by masoom** [rancher/rancher#21159]
**What kind of request is this (question/bug/enhancement/feature request):**
Bug

**Steps to reproduce (least amount of steps as possible):**
1. Setup Rancher HA cluster running v2.2.4
2. Configure Active Directory authentication
3. Upgrade Rancher to v2.2.4
4. Go to the Rancher Project, click on Members
5. The ldap group for the Member is missing

**Result:**

**Other details that may be helpful:**

We did a rancher upgrade to `v2.2.4` and also changed the ldap system user used in Rancher to make `ldap` log-in(authentication) work again.

We do not see `role binding` in Rancher UI but at `projectroletemplatebindings` CRD objects, we can see the role bindings.  At kubernetes level, we still see role binding between LDAP group to the defined role.

When we go to the Project, We see the users and system users but don't see the ldap groups.

When we create a new project and add new `ldap group` to a `role`, it does not show that the ldap group was added in the UI.

**Environment information**
- Rancher version : `v2.2.4`

- Installation option (single install/HA): `HA` 

**Cluster information**
- Cluster type (Hosted/Infrastructure Provider/Custom/Imported): Hosted
- Machine type (cloud/VM/metal): `VM` 
- Kubernetes Client version (use `kubectl version`): v1.12.7" 
- Kubernetes Server version (use `kubectl version`): v1.12.3"

```
Client Version: version.Info{Major:"1", Minor:"12", GitVersion:"v1.12.7", GitCommit:"6f482974b76db3f1e0f5d24605a9d1d38fad9a2b", GitTreeState:"clean", BuildDate:"2019-03-25T02:52:13Z", GoVersion:"go1.10.8", Compiler:"gc", Platform:"linux/amd64"}
Server Version: version.Info{Major:"1", Minor:"12", GitVersion:"v1.12.3", GitCommit:"435f92c719f279a3a67808c80521ea17d5715c66", GitTreeState:"clean", BuildDate:"2018-11-26T12:46:57Z", GoVersion:"go1.10.4", Compiler:"gc", Platform:"linux/amd64"}
```

- Docker version (use `docker version`):

```
(paste the output here)
```

**Comment by canhnt** [comment]
It's similar to https://github.com/rancher/rancher/issues/20760

**Comment by mrajashree** [comment]
Dup of #20760 
@masoom It's fixed in 2.2.5 https://github.com/rancher/rancher/issues/20825
Please give that a try and open a new issue if the problem persists
